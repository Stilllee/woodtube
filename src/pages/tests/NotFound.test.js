import NotFound from "../NotFound";
import React from "react";
import { Route } from "react-router-dom";
import { render } from "@testing-library/react";
import { withRouter } from "../../tests/utils";

describe("NotFound", () => {
  it("올바르게 렌더링", () => {
    const { asFragment } = render(
      withRouter(<Route path="/" element={<NotFound />} />)
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
