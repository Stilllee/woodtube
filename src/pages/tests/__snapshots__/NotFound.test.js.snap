// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound 올바르게 렌더링 1`] = `
<DocumentFragment>
  <div>
    Not Found!
  </div>
</DocumentFragment>
`;
